torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -680,9 +681,10 @@ def call_function(
 
         # Specialize into one of the branches since pred is constant
         if type(args[0]) is ConstantVariable:
-            log.warning(
-                "Pred is a Python constant. When used with torch.cond, it executes only one of the branches."
-                " If you want torch.cond to perserve two branches, please make the predicate a boolean tensor or a SymBool."
+            warnings.warn(
+                "Pred is a Python constant. When exporting torch.cond, it exports one of the branches."
+                " If you want torch.cond to perserve two branches, please make the predicate a boolean tensor or a SymBool.",
+                UserWarning,
```

Comment:
The previous error message was better? Since this is the compile path, not the export path.

We should say that we're "specializing" on one of the branches. Also "perserve -> preserve".